src/OrchardCore.Modules/OrchardCore.Flows/Views/FlowPartSettings.Edit.cshtml

```diff
@@ -11,3 +12,13 @@
     <label class="form-check-label" asp-for="CollapseContainedItems">@T["Collapse Contained Items"]</label>
     <span class="hint">@T["When checked, the contained items will be collapsed when the edit form is loaded."]</span>
 </div>
+<div class="mb-3">
+    <label asp-for="DefaultAlignment" class="form-label">@T["What default alignment should be used?"]</label>
+    <select asp-for="DefaultAlignment" class="form-select" id="type-part-editor-select">
+        <option value="">@T["Please choose an alignment"]</option>
+        @foreach (var alignment in Enum.GetValues(typeof(FlowAlignment)))
+        {
+            <option value="@alignment">@T[alignment.ToString()]</option>
```

Comment:
This localized string can't be extracted for translation by the POExtractor. You need to pass a literal string to `@T[]`.